src/bust.py

```diff
@@ -327,125 +327,103 @@ def get_google_form_url(self, image_url: Optional[str] = None) -> Optional[str]:
+            index + 1,
+            submit_message.author.mention,
+            song_utils.song_format(local_filepath, attachment.filename),
+            attachment.url,
+            submit_message.jump_url,
         )
 
-        # Break on no songs to list
-        if len(channel_media_attachments) == 0:
-            await message.channel.send("There aren't any songs there.")
-            return None
-
-        bc = BustController(client, channel_media_attachments, target_channel)
+        # We only add the embed description prefix to the first message
+        description_prefix_charcount = 0
+        if len(embed_description_list) == 0:
+            description_prefix_charcount = len(embed_description_prefix)
+
+        if (
+            description_prefix_charcount
+            + len(embed_description_current)
+            + len(song_list_entry)
+            > config.EMBED_DESCRIPTION_LIMIT
+        ):
+            # If adding a new list entry would go over, push our current list entries to a new embed
+            embed_description_list.append(embed_description_current)
+            # Start a new embed
+            embed_description_current = song_list_entry
+        else:
+            embed_description_current += song_list_entry
 
-        # Title of !list embed
-        embed_title = "❤️‍🔥 AIGHT. IT'S BUSTY TIME ❤️‍🔥"
-        embed_description_prefix = "**Track Listing**\n"
+    # Add the leftover part to a new embed
+    embed_description_list.append(embed_description_current)
 
-        # List of embed descriptions to circumvent the Discord character embed limit
-        embed_description_list: List[str] = []
-        embed_description_current = ""
+    # Iterate through each embed description, send and pin messages
+    message_list = []
 
-        for index, (
-            submit_message,
-            attachment,
-            local_filepath,
-        ) in enumerate(channel_media_attachments):
-            list_format = "**{0}.** {1}: [{2}]({3}) [`↲jump`]({4})\n"
-            song_list_entry = list_format.format(
-                index + 1,
-                submit_message.author.mention,
-                song_utils.song_format(local_filepath, attachment.filename),
-                attachment.url,
-                submit_message.jump_url,
+    # Send messages, only first message gets title and prefix
+    for index, embed_description in enumerate(embed_description_list):
+        if index == 0:
+            embed = Embed(
+                title=embed_title,
+                description=embed_description_prefix + embed_description,
+                color=config.LIST_EMBED_COLOR,
+            )
+        else:
+            embed = Embed(
+                description=embed_description,
+                color=config.LIST_EMBED_COLOR,
             )
+        list_message = await interaction.channel.send(embed=embed)
+        message_list.append(list_message)
+
+    # If message channel == target channel, or we ask for a mock,
```

Comment:
```suggestion
    # If message channel == target channel,
```

Since we removed this functionality for the time being.